### PR TITLE
Teiid Docker image

### PR DESCRIPTION
**Pull request by cvanball**

### CODE REVIEW

**Comment by paulrobinson** [comment]
this is ok to test

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
I'm getting 404s on http://www-stg.jboss.org/pr/1025/build/2336/docker/ for the Teiid addition. Is that expected?

**Comment by cvanball** [comment]
No expected. Looks like the Teiid image not uploaded yet. Please check with Ramesh, he had the latest image available,

**Comment by paulrobinson** [comment]
I'm not familiar with who 'Ramesh' is. Please let me know when this PR is ready to go, and I'll review again.

**Comment by cvanball** [comment]
Hi Paul,

The docker file and image are located here.
https://hub.docker.com/r/jboss/teiid/

Met vriendelijke groet/Kind regards,

Red Hat BV


Cojan van Ballegooijen
Senior Solutions Architect
Building The Entree II
De Entree 238
1101 EE Amsterdam
tel: +31 20 5651225
mob: +31 6 21261996
email: cojan@redhat.com

**Comment by paulrobinson** [comment]
Thanks. Looks like we'll need a modification to the template to support links that are not in the JBoss Docker GitHub org.

@adelasofia can you take this PR and add the ability to link to https://hub.docker.com/r/jboss/teiid/ . I'll comment in the code, to be more specific.

File: _config/docker_images.yml [hunk 1]
```diff
@@ -135,3 +135,12 @@
...
       path: artificer/blob/master/artificer-wildfly
       description: Artificer on Wildfly
+
+- name: Teiid
+  description: Teiid is a data virtualization system that allows applications to use data from multiple, heterogenous data stores.
+  url: http://teiid.jboss.org
+  image_url: http://static.jboss.org/teiid/images/teiid_200x150.png
+  images: 
+    - name: jboss/teiid-wildfly
+      path: teiid/blob/master/teiid-wildfly
```
Comment (paulrobinson) [inline _config/docker_images.yml]: I think we need an alternative for this, that allows linking to: https://hub.docker.com/r/jboss/teiid/. Maybe a key called 'url'?

**Comment by paulrobinson** [comment]
Sorry, this fell off my Radar. I'll ping @adelasofia to take a look.

File: _config/docker_images.yml (hunk 1 above)
Comment (adelasofia) [inline _config/docker_images.yml]: @paulrobinson I don't quite understand what this is... Where should this 'url' variable be called from?

Comment (paulrobinson) [inline _config/docker_images.yml]: Maybe I was confusing matters by hinting at implementation details, The fundamental problem is that the links are broken for Teiid. Can you update the PR to fix them?

**Comment by adelasofia** [comment]
@cvanball Please merge this PR https://github.com/cvanball/www.jboss.org/pull/1

**Comment by cvanball** [comment]
Done

**Pull request closed without merging**